### ISSUE TITLE
missing style.css file? result in no offer sent??

### ISSUE THREAD

**Issue by gilav** [Shadid12/live-video-tut#1]
Hello

I follow your tutorial and found it interesting (I already did some chat app, but wanted to try the video part).

The style.css file is missing from the repository. I don't know if it is because of this but the video connection don't start. 

Can you provide it ?

Thanks

Gilles Lavaux

**Comment by rachiitgupta** [comment]
Yeah same issue. I even get another error in the console, saying 'Server responded with error 404'!
